website_sale_attribute_filter_category/tests/test_website_sale_attribute_filter_category.py

```diff
@@ -15,7 +15,6 @@ def setUp(self):
         self.product_attribute = ProductAttribute.create(
             {
                 "name": "Test",
-                "website_published": True,
```

Comment:
Why not published?

Reply:
The field is_published has the default value True

Reply:
In this case I prefer to be `explicit better than implicit`. If your test depends on this value, don't rely on default values, that other modules or versions can change it.

Reply:
Ok, so I have to see it better, because this field is giving me problems on test...